packages/modules-runtime/server.js

```diff
@@ -15,6 +15,14 @@ makeInstallerOptions.fallback = function (id, parentId, error) {
   if (topLevelIdPattern.test(id)) {
     if (typeof Npm === "object" &&
         typeof Npm.require === "function") {
+      if (id && id.startsWith('meteor/')) {
```

Comment:
This is looking better, but I think this check should be hoisted outside the `if (typeof Npm === ...` block, since the new error is relevant even if `Npm.require` is not defined?